src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,15 @@
 import React from 'react';
 import './MoviesList.scss';
+import moviesFromServer from '../../api/movies.json';
```

Comment:
you should import moviesFromServer in App.js and then pass them to MovieList component